signingscript/docker.d/passwords.yml

```diff
@@ -78,42 +78,6 @@ in:
              {"$eval": "AUTOGRAPH_REFERENCE_BROWSER_PASSWORD"},
              ["autograph_apk"]
           ]
-        project:mobile:firefox-android:releng:signing:cert:dep-signing:
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_GPG_USERNAME"},
-             {"$eval": "AUTOGRAPH_GPG_PASSWORD"},
-             ["autograph_gpg"]
-          ]
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FOCUS_USERNAME"},
-             {"$eval": "AUTOGRAPH_FOCUS_PASSWORD"},
```

Comment:
init_worker.sh should be updated to not require variables we're no longer using.

Reply:
Done. Also noticed/remembered pushapkscript.